styles.css

```diff
@@ -21,6 +24,44 @@ header {
   flex-direction: column;
 }
 
+.mobile-nav {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  transition: 0.2s ease-in-out;
+  background-color: rgba(41, 58, 208, 0.8);
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
+  display: none;
+}
```

Comment:
- [OPTIONAL] It would be nice if when the menu is open the user cannot scroll down or the menu follows the user's scrolling behavior. Now it is possible to scroll down when the menu is open creating a bad user experience. 
 - 
<img width="482" alt="image" src="https://user-images.githubusercontent.com/61919617/223764124-9cb1ed25-2e07-4bb3-bd1d-11835c3188ea.png">